code/game/objects/structures/displaycase.dm

```diff
@@ -14,6 +14,9 @@
 	var/alert = TRUE
 	var/open = FALSE
 	var/openable = TRUE
+	var/security_level_locked = FALSE //For locking the case to different access levels based on whether the station is at an elevated security level
+	var/security_level_access = ACCESS_SECURITY	//Access level required after security level elevates
+	var/green_level_access	//for storing the original access requirements
```

Comment:
Those 3 could just be one var

Reply:
Done and also simplified my overly complicated If statements.